jabkit/src/main/java/org/jabref/JabKit.java

```diff
@@ -77,7 +77,7 @@ public static void main(String[] args) {
             // Process arguments
             ArgumentProcessor argumentProcessor = new ArgumentProcessor(preferences, entryTypesManager);
             CommandLine commandLine = new CommandLine(argumentProcessor);
-            String usageHeader = String.format(ArgumentProcessor.JABREF_BANNER, new BuildInfo().version);
+            String usageHeader = String.format(BuildInfo.JABREF_BANNER, new BuildInfo().version);
```

Comment:
JabKit show `JabKit` IMHO - not `JabRef`

Reply:
JabKit is IMHO part of JabRef, it's just the command line interface to jablib. Keep the well known name. Can be mentioned in a subtitle.

Reply:
A compromise:
![grafik](https://github.com/user-attachments/assets/41168ea3-c071-4f25-9c54-526fdf5c22de)